.coderabbit.yaml

```
@@ -1,212 +1,114 @@
+    
+  # 리뷰 대상 파일 패턴 (포함할 파일들)
   include_patterns:
-    - "src/**/*.{js,ts,jsx,tsx}"
-    - "lib/**/*.py"
-    - "**/*.go"
-    - "test-files/**/*.js"  # 테스트용 파일 포함
-    - "*.{md,yml,yaml,json}"
+    - "**/*.{js,ts,jsx,tsx}"     # JavaScript/TypeScript 파일
+    - "**/*.{py,java,go,cpp,c,cs}"  # 기타 프로그래밍 언어
+    - "**/*.{yml,yaml,json,md}"   # 설정 파일 및 문서
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)